pkg/utils/utils.go

```diff
@@ -140,18 +140,19 @@ func GetSharedPF(ifName string) (string, error) {
 
 // GetVFLinkNames returns VF's network interface name given it's PF name as string and VF id as int
 func GetVFLinkNames(pfName string, vfID int) ([]string, error) {
-	var names []string
+	var names = make([]string, 0)
+
 	vfDir := filepath.Join(NetDirectory, pfName, "device", fmt.Sprintf("virtfn%d", vfID), "net")
 	if _, err := os.Lstat(vfDir); err != nil {
-		return nil, err
+		// if net directory is missing the device is probably unregistered from its network driver
+		return names, nil
```

Comment:
With netdevice we need be able to catch this error.  As @zshi-redhat indicated, this is going to be a temp fix for now, and need to be reverted back with newer dpdk support.

Reply:
I actually need to check which driver the VF is attached to, and skip for vfio / uio only.